crypto/fipsmodule/rsa/rsa.c

```diff
@@ -878,12 +1026,32 @@ int rsa_private_transform(RSA *rsa, uint8_t *out, const uint8_t *in,
 
 int RSA_flags(const RSA *rsa) {
   SET_DIT_AUTO_DISABLE;
+  if (rsa == NULL) {
+    OPENSSL_PUT_ERROR(RSA, ERR_R_PASSED_NULL_PARAMETER);
+    return 0;
+  }
+
   return rsa->flags;
 }
 
+void RSA_set_flags(RSA *rsa, int flags) {
```

Comment:
NP: Do we also need `RSA_clear_flags`?

Reply:
RSA related functionality, don't need as of now